prompt2model/dataset_generator/prompt_based.py

```diff
@@ -147,14 +147,14 @@ def construct_prompt(
                 parsed from the user's prompt, which quality is higher than the
                 generated examples.
             generated_examples: A list of currently generated examples.
-            context_cutoff: If the total length of the prompt exceeds this value,
-                repeat the prompt generation process to generate a shorter one.
+            context_cutoff: If the total length of the prompt in tokens exceeds this
+                value, repeat prompt generation process to generate a shorter one.
```

Comment:
Suggestion to make grammatical (while still fitting within char limits)

```suggestion
                value, repeat the prompt generation process to generate a shorter one.
```